modules/fargate/variables.tf

```diff
@@ -0,0 +1,56 @@
+variable "cluster_name" {
```

Comment:
When the cluster is disabled (create_eks = false) all other variables should not be required. Give the default values to all of them, and add a simple module block for the disabled in the example.

Reply:
I added some default to the submodule even their are all coming from the root module. So if create_eks is false, all variable in this submodule will be set correctly.

This fargate submodule is not intended to be used alone.